aws/lambdas/getMatchDetails/index.js

```diff
@@ -0,0 +1,232 @@
+'use strict';
+
+const http      = require('http');
+const cheerio   = require('cheerio');
+const _         = require('lodash');
+
+let $;
+
+const requestOptions = {
+    host: 'www.uzletiliga.hu',
+    path: '/eredmenyek/match_details3.php'
+};
+
+const getQueryParams = (element) => {
+    if (!$(element).prop('href')) {
+        throw new Error('No href attribute');
+    }
+
+    return $(element).prop('href').split('?')[1].split('&');
+};
+
+const addQueryParams = (queryParams, requestOptions) => {
+    requestOptions.path += Object.keys(queryParams).reduce((queryParamsString, paramKey, paramIndex) => {
+        if (paramIndex === 0) {
+            queryParamsString += '?';
+        } else {
+            queryParamsString += '&';
+        }
+
+        return queryParamsString += `${paramKey}=${queryParams[paramKey]}`;
+    }, '');
+    
+    return requestOptions;
+};
+
+const parseMatchPage = (html, matchId, eventId) => {
+    $ = cheerio.load(html);
+
+    const matchChronology = $('.match_details_table').filter((index, element) => {
+        return $(element).find('h6').text() === 'Meccs kronológia';
+    });
+
+    const matchTable = $('.match_details_table').filter((index, element) => {
+        return $(element).find('h6').text() === 'Meccslap';
+    });
+
+    const date = matchTable.find('tr').filter(function () {
+        return $(this).find('td').eq(0).text() === 'Dátum';
+    }).find('td').eq(1).text().split(' ').join(':');
+
+    const location = matchTable.find('tr').filter(function () {
+        return $(this).find('td').eq(0).text() === 'Helyszín';
+    }).find('td').eq(1).text();
+
+    const round = getRound(matchTable.find('tr').filter(function () {
+        return $(this).find('td').eq(0).text() === 'Szakasz';
+    }));
+
+    const group = matchTable.find('tr').filter(function () {
+        return $(this).find('td').eq(0).text() === 'Csoport';
+    }).find('td').eq(1).text().split('-')[0];
+
+    const { host, path } = addQueryParams({ eid: eventId, mid: matchId }, Object.assign({}, requestOptions));
+
+    const matchDetailsLink = host + path;
+
+    let homeTeam = {
+        id: '',
+        players: []
+    };
+
+    let awayTeam = {
+        id: '',
+        players: []
+    };
+
+    getPlayers(homeTeam, awayTeam);
+
+    getStats(matchChronology, homeTeam, awayTeam);
+
+    return {
+        homeTeam,
+        awayTeam,
+        id: matchId,
+        date,
+        location: {
+            name: location
+        },
+        group,
+        round,
+        matchDetailsLink
+    };
+
+    function getMatchDetailsLink(eventId, matchId) {
```

Comment:
Could we agree of a coding style that we never put functions behind return statement, I am pretty sure that you learned it from someone we both know and work(ed) with but it is not very clean code :)